src/Publishing/PublishingProject.cs

```diff
@@ -0,0 +1,78 @@
+//-----------------------------------------------------------------------
+// <copyright file="PublishingProjectModel.cs" company="GlobalLink Vasont">
+// Copyright (c) GlobalLink Vasont. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+namespace Vasont.Inspire.Models.Publishing
+{
+    /// <summary>
+    /// This model class represents the project detail for a publishing project.
+    /// </summary>
+    public class PublishingProjectModel
+    {
+        /// <summary>
+        /// Gets or sets the project identity.
+        /// </summary>
+        public long ProjectId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the project title.
+        /// </summary>
+        public string Title { get; set; }
+
+        /// <summary>
+        /// Gets or sets the description of the project.
+        /// </summary>
+        public string Description { get; set; }
+
+        /// <summary>
+        /// Gets or sets the component identity.
+        /// </summary>
+        public long ComponentId { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the project should be validated.
+        /// </summary>
+        public bool SkipValidation { get; set; } = false;
+
+        /// <summary>
+        /// Gets or sets the filter condition for the project.
+        /// </summary>
+        public string Filter { get; set; }
+
+        /// <summary>
+        /// Gets or sets the export configuration for the project.
+        /// </summary>
+        public string ExportConfig { get; set; }
```

Comment:
What are you storing here?  Should this be the export configuration id?

Reply:
It could be.  Is there an export configuration table that should be related? Is there a corresponding model?  I had not found one so I went with raw data